docs/pull_request_template.md

```diff
@@ -0,0 +1,61 @@
+# Description
+
+Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.
+
+You can refer to [Kubernetes community documentation] on writing good commit messages, which provides good tips and ideas.
+
+Some PRs addess specific issues. Please, refer to the [CONTRIBUTING] documentation for more
+information on how to link a PR to an existing issue.
+
+# What environments does this code impact?
+
+- [ ] Cloud
+- [ ] Operator Managed Deployments
+- [x] None
+
+
+# How was this code tested?
+
+Please, select one or more if needed:
+
+- [ ] subsystem tests
```

Comment:
I think that if someone review he will see that subsystem was added a well, so i'm not sure if it's needed. 
I think that all this section is more related to the review guidelines and not part of the PR.

Reply:
I have intended this section as a way for the author to communicate how the PR was tested. In some cases subsystem is enough (in this case there is probably a subsystem test added but in some cases, the current tests may already cover the PR).